### ISSUE TITLE
Videos fail to appear when wanting to share within an app

### ISSUE THREAD

**Issue by Flam3z** [SimpleMobileTools/Simple-Gallery#1828]

**Comment by tibbi** [comment]
Im not sure what you mean, tell some steps how to reproduce it please. What app version are you using?

**Comment by Flam3z** [comment]
Version 6.14.8

I wanted to share a video externally instead of from what the app's image picker would give me so I went to the gallery app and I couldn't find any videns even in the folders which I knew videos were in.

**Comment by tibbi** [comment]
well why would some Image Picker show videos?

**Comment by Flam3z** [comment]
Some app allow videos to be shown or categories it by gifs, images or videos like telegram in a way.

**Comment by tibbi** [comment]
Tell exact steps to reproduce please. You press Add File in some app?

**Comment by Flam3z** [comment]
Correct, then a prompt comes up asking me which gallery app I want to open and i obviously chose simple gallery.

**Comment by tibbi** [comment]
but in what app are you pressing Add File? I dont see any custom gallery selection in Telegram, I see only the built in one

**Comment by tibbi** [comment]
but in what app are you pressing Add File? Adding file to Telegram works fine to me, I see videos too

**Comment by Flam3z** [comment]
I was using discord when this happened and they have a feature where you can fetch a photo or file externally from another app.

**Comment by tibbi** [comment]
ok thanks, I will try reproducing it like that

**Comment by Flam3z** [comment]
Thank you, sorry if i caused any confusion.

**Comment by 9p6** [comment]
Discord apart from having internal media browser also has ability to choose file for sharing through other apps. The problem is that Simple Gallery filters out videos when launched this way. This is not the case when using other apps like e.g. LineageOS default gallery app on the right. This is likely relevant not only to Discord but also other apps with file chooser. Anyway it would great to be able access all media!

![Imagepipe_0](https://user-images.githubusercontent.com/74108881/122410939-a5d89780-cf84-11eb-94d6-67a62e80ff5e.png)

**Comment by cunninglemming** [comment]
Would be great to have this one fixed
